src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -1210,6 +1173,47 @@ private ObTableBatchOperation buildObTableBatchOperation(List<KeyValue> keyValue
         return batch;
     }
 
+    private com.alipay.oceanbase.rpc.mutation.Mutation buildMutation(KeyValue kv, boolean putToAppend) {
+        KeyValue.Type kvType = KeyValue.Type.codeToType(kv.getType());
+        switch (kvType) {
+            case Put:
+                ObTableOperationType operationType;
+                if (putToAppend) {
+                    operationType = APPEND;
+                } else {
+                    operationType = INSERT_OR_UPDATE;
+                }
+                return com.alipay.oceanbase.rpc.mutation.Mutation.getInstance(operationType, ROW_KEY_COLUMNS,
+                        new Object[]{kv.getRow(), kv.getQualifier(), kv.getTimestamp()}, V_COLUMNS,
+                        new Object[]{kv.getValue()});
+            case Delete:
+                return com.alipay.oceanbase.rpc.mutation.Mutation.getInstance(DEL, ROW_KEY_COLUMNS,
+                        new Object[]{kv.getRow(), kv.getQualifier(), kv.getTimestamp()}, null, null);
+            case DeleteColumn:
+                return com.alipay.oceanbase.rpc.mutation.Mutation.getInstance(DEL, ROW_KEY_COLUMNS,
+                        new Object[]{kv.getRow(), kv.getQualifier(), -kv.getTimestamp()}, null, null);
+            case DeleteFamily:
+                return com.alipay.oceanbase.rpc.mutation.Mutation.getInstance(DEL, ROW_KEY_COLUMNS, new Object[]{kv.getRow(), null, -kv.getTimestamp()},
```

Comment:
Will mutation handle null in rowkey well?

Reply:
yes, 
![image](https://github.com/oceanbase/obkv-hbase-client-java/assets/34029771/54c80ac5-0bf8-46cf-becd-5c2fb0486183)